_posts/2023-05-12-sprint-attendees.md

```diff
@@ -0,0 +1,49 @@
+layout: post
+published: true
+title: "Sprints: What are they & why should I attend?"
+---
+
+(In the ancient days of FLOSS…) Development sprints are most common around Free/Libre Open Source Software, or FLOSS projects. Most of these projects (like Django and Python) are maintained by a community of volunteers. These volunteers usually work together while spread around the world. Conferences present a rare occasion for these developers to meet in person. And thus, sprints were born! In the context of a given conference or event, you’ll often hear them referred to as “the sprints” or just “sprints”.
+
+Over time, these core developers realized sprints also provided them an opportunity to help find new potential contributors. And that’s where YOU come in.
+
+## Why Attend a Sprint at DjangoCon Europe 2023?
+Sprints give you an opportunity to work shoulder-to-shoulder with experts in the community. In addition, there are practical and personal benefits to attending the sprints.
+
+From a practical perspective, if your organisation makes heavy use of Django or a related package, getting involved in its maintenance will give you a better understanding of that package, as well as its strengths and limitations. On the personal side, most people won’t usually work on a project that’s as widely used as Django or some of the other related packages. Getting involved in the maintenance of one of these packages can expose you to engineering practices you wouldn’t see otherwise. This can help you become a better engineer while allowing you to contribute to the community that benefits you and your company.
+
+
+## Questions and Answers
+### I’ve never taken part in a sprint before though. So can I join in?
+
+Absolutely. Everyone’s welcome. It’s not an elite club, or for experts only. All of the projects that will be running sprints welcome anyone who wants to help.
+
+### What do I need?
+
+You'll need to know the basics of using Git/GitHub to check out code and make pull requests, to know how to use the issue tracker, and so on. You’ll also need your own computer with a suitable Python environment set up on it - but there'll be people to help with that too.
+
+If you are new to Git, we recommend the first three chapters of [Pro Git](https://git-scm.com/book/en/v2), which is freely available to read online or to download. Those chapters will get you started and give you about 80% of the Git you’ll use most often. Chapters five and six, on Distributed Git and Github respectively, are also very helpful if you’re new to working with a group on software.
+
+### Do I have to stay for a whole sprint?
+
+Not at all. Drop in just for a bit, if you just want to see what it’s like.
+
+### Is it needed to good Python programmer?
+
+You truly don’t need to be very good Python programmer. In fact, you barely need to be a programmer at all, because even the most novice sprinter can make a very valuable contribution by helping improve documentation, for example.
```

Comment:
```suggestion
You truly don’t need to be a very good Python programmer. In fact, you barely need to be a programmer at all, because even the most novice sprinter can make a very valuable contribution by helping improve documentation, for example.
```